### PR TITLE
[Vis colors] Update legacy mapped colors in charts plugin to use ouiPaletteColorBlind()

### PR DESCRIPTION
**Pull request by manasvinibs**
### Description

In this change, we are deleting legacy color palette file used to generate mapped colors and instead added OUI palette colors. Also, updated the default color in legacy visualizations to use `ouiPaletteColorBlind()[0]`.

### Issues Resolved

https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4320
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4323

## Screenshot

Area:
Screenshot

Pie:
Screenshot

Vertical bar:

Screenshot

Horizontal bar:

Screenshot

Single series:

Screenshot

Screenshot

Screenshot3

vertical bar

### Check List

- [X] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
  - [ ] `yarn test:ftr`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [X] Update [CHANGELOG.md](./../CHANGELOG.md)
- [X] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by joshuarrrr — CHANGES_REQUESTED** [review]
Ah, interesting, these screenshots are very helpful - we _do_ want to rotate in both directions, but we may need to adjust the ordering of the colors, because we preferentially want to start with the middle colors, not the darkest.

**Comment by manasvinibs** [comment]
> 



> Ah, interesting, these screenshots are very helpful - we _do_ want to rotate in both directions, but we may need to adjust the ordering of the colors, because we preferentially want to start with the middle colors, not the darkest.

I've adjusted the ordering of the color along with sort criteria. Can you take a look into screenshots now?

File: src/plugins/charts/public/services/colors/mapped_colors.ts [hunk 1]
```diff
@@ -100,7 +100,13 @@ export class MappedColors {
 
     // Generate a color palette big enough that all new keys can have unique color values
     const allColors = _(this._mapping).values().union(configColors).union(oldColors).value();
-    const colorPalette = createColorPalette(allColors.length + keysToMap.length);
+    const numColors = allColors.length + keysToMap.length;
+    const colorPalette = euiPaletteColorBlind({
+      rotations: Math.ceil(numColors / 10),
+      direction: 'both',
+      order: 'group',
```
Comment (joshuarrrr) [inline src/plugins/charts/public/services/colors/mapped_colors.ts:107]: I think we actually want to use 
```suggestion
      order: 'append',
```

The idea is that the first 10 colors assigned should be the default (non-rotated). Then the next 10 (11-20) come from one of the first rotations, in the same order. Then the next 10 (21-30) come from the first rotation in the other direction. I think we may also want to remove the `natural` sort to get them ordered correctly.

Comment (manasvinibs) [reply src/plugins/charts/public/services/colors/mapped_colors.ts]: Yes, I think the screenshot was misleading before. I've updated the screenshot after adjusting the order and sort criteria.

File: test/functional/apps/dashboard/dashboard_state.js [hunk 2]
```diff
@@ -278,13 +278,13 @@ export default function ({ getService, getPageObjects }) {
           await retry.try(async () => {
             const pieSliceStyle = await pieChart.getPieSliceStyle('80,000');
             // The default green color that was stored with the visualization before any dashboard overrides.
-            expect(pieSliceStyle.indexOf('rgb(87, 193, 123)')).to.be.greaterThan(0);
+            expect(pieSliceStyle.indexOf('rgb(84, 179, 153)')).to.be.greaterThan(0);
```
Comment (manasvinibs) [inline test/functional/apps/dashboard/dashboard_state.js:281]: I've created [this](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4424) issue for removing hard coded color values from functional tests.

**Review by BSFishy — APPROVED** [review]

**Review by joshuarrrr — APPROVED** [review]
This can be a follow-up task, but it would be nice to update https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/docs/charts/current_usage.md now that we've changed quite a bit of the usage.

**Review by ashwin-pc — APPROVED** [review]

**Pull request merged**